2-mandatory/3-bush-berries.js

```diff
@@ -22,7 +22,12 @@
 */
 
 function isBushSafe(berryArray) {
-  //Write your code here
+  let safeToEat = berryArray.every((color) => color === "pink");
+  if (safeToEat) {
+    return "Bush is safe to eat from";
+  } else {
+    return "Toxic! Leave bush alone!";
+  }
```

Comment:
Nice one! 
Two things to do if you're interested to learn more!
1- using some() and every() method together as the instruction says
2- when you have if else statement you remove else, and just return what is supposed to be in else. Please research about it. It cleans up your code!